src/test/java/com/mongodb/hibernate/SessionFactoryTests.java

```diff
@@ -50,22 +52,33 @@ void testInvalidConnectionString() {
 
     @Test
     void testOpenSession() {
-        try (var sessionFactory = buildSessionFactory(Map.of())) {
-            Assertions.assertDoesNotThrow(() -> sessionFactory.openSession().close());
+        try (var sessionFactory = buildSessionFactory()) {
+            assertDoesNotThrow(() -> sessionFactory.openSession().close());
         }
     }
 
+    @Test
+    void testInTransaction() {
+        try (var sessionFactory = buildSessionFactory()) {
+            sessionFactory.inTransaction(session -> {});
+        }
+    }
```

Comment:
some code refactoring was done in existing `SessionFactoryTests` so no need of empty map to be passed in default scenario.